Dockerfile

```diff
@@ -3,14 +3,10 @@ FROM ubuntu:22.04
 ENV TZ=Etc/UTC
 RUN ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && echo $TZ > /etc/timezone
 
-RUN apt-get update
-RUN apt-get -y install bash git cmake python3 python3-pip python3-virtualenv python3-cryptography python3-future python3-click python3-serial python3-wheel python3-pyparsing python3-pyelftools
+RUN apt-get update && apt-get -y install bash git cmake python3 python3-pip python3-virtualenv python3-cryptography python3-future python3-click python3-serial python3-wheel python3-pyparsing python3-pyelftools python-is-python3 libncurses-dev flex bison gperf
```

Comment:
Just a nitpick, but I think this long script is bit unreadable. Can you please do the `&& \<newline>` to format it better.

Reply:
Fixed in c2d03d9b8b299ef0c71d9928c87b7066ebb20e23